.isort.cfg

```diff
@@ -0,0 +1,15 @@
+[settings]
+py_version=310
+skip=.gitignore, __init__.py
+skip_glob=docs/*, .github/*
+line_length = 88
```

Comment:
I think we want 120 instead of 88 to align with the other parts of the code.

Reply:
I had kept this to 88 because if isort and black are used together, their irregular line lengths might cause issues later. 88 seemed like the default because that is what black uses. Changing it to 120 for now. 

This restriction by black might need to added to gitignore later after #180 is closed or perhaps black's line length could be set to 120 in #180.